dbt/models/shared_columns.md

```diff
@@ -186,6 +186,17 @@ Reason for change or no change
 
 # Assessed / Market Values
 
+## alternative_cdu
+
+{% docs shared_column_alternative_cdu %}
+Condition/Desirability/Utility code.
+
+Code representing any number of seemingly unrelated characteristics
+associated with a PIN, ranging from condition to types of subsidies, to
+whether or not a PIN is a garage. The full list of CDU codes can be found on
+the Assessor's site. For "condition", see the `cdu` column.
```

Comment:
**[Suggestion, non-blocking]** We _could_ add a direct link here, but I'm open to leaving it out if we think e.g. the link has a high chance of breaking in the future.

Reply:
I don't actually think they exist on the website.